src/main/docs/guide/kafkaListener/kafkaErrors.adoc

```diff
@@ -46,6 +46,14 @@ NOTE: Specify exception to retry apply only for RETRY_ON_ERROR error strategy.
 
 When an exception occurs in a ann:configuration.kafka.annotation.KafkaListener[] method by default the exception is simply logged. This is handled by api:configuration.kafka.exceptions.DefaultKafkaListenerExceptionHandler[].
 
+[TIP]
+====
+DefaultKafkaListenerExceptionHandler behavior can be configured via methods:
+
+- `setSkipRecordOnDeserializationFailure`: Sets whether the seek past records that are not deserializable (enabled by default).
```

Comment:
I think we should create a configuration object and let users configure this in `application.properties`

Reply:
@sdelamo Done 366528a1b68b2b7647c52e6542884b5391d7dadc